prosemirror-jdita/src/schema.ts

```diff
@@ -3,16 +3,36 @@ import { getDomNode } from './dom';
 import { ChildTypes } from 'jdita';
 import { NodeSpec, Schema, SchemaSpec, Node, MarkSpec, DOMOutputSpec } from 'prosemirror-model';
 
-
+/**
+ * Set the root node `document` to string "doc"
+ *
+ * @remarks
+ * Will be used in `defaultNodeName()`.
+ */
 export const NODE_NAMES: Record<string, string> = {
   document: 'doc',
 }
-export const TO_DOM: Record<string, (node: typeof BaseNode, attrs: any) => (node: Node) => DOMOutputSpec> = {
-}
+
+/**
+ * Provide a map of special nodes to their corresponding DOM node
+ */
+// TODO: Why is this is empty?
```

Comment:
TODO needs to be resolved.

Reply:
I have resolved this comment and moved the TODO as an open question to our internal list (56e4f7a).

Reply:
This question has been moved to a list on the Petal google document, and the `TODO` has been removed from the code.

Reply:
This question has been moved to a list on the Petal google document, and the `TODO` has been removed from the code.